app/controllers/concerns/my_facilities_filtering.rb

```diff
@@ -18,6 +18,10 @@ def filter_facilities
       facilities_by_size(filtered_facilities)
     end
 
+    def facilities_for_group
+      facilities_by_facility_group(@accessible_facilities)
```

Comment:
Instead of introducing a new method, can you call this directly inline in the controller?  This method isn't really carrying its weight ...and long term I think we should be moving away from this module based filtering approach.  

Something like this in the controller:

```
facilities_for_totals = facilities_by_facility_group(@accessible_facilities)
```

This would make it more clear that we are constructing totals based on a superset of what is being filtered and shown in the view.

Reply:
Yeah, sounds good.